rj_gameplay/stp/skill/__init__.py

```diff
@@ -5,74 +5,32 @@
 from typing import Dict, List, Type, TypeVar
 
 import stp.role as role
+import stp.rc
+from rj_msgs.msg import RobotIntent
 
 
 class ISkill(ABC):
```

Comment:
Is there any need for theses interfaces, if not we can delete all of them.

Reply:
they exist so legacy code doesn't brick the compiler, same as RoleRequest et al